tests/branching.py

```diff
@@ -38,6 +40,13 @@
 """
 DO NOT CHANGE ANYTHING BELOW HERE
 """
+# If csv must be created, create an empty file here
+if create_csv:
+    csv_filepath = create_csv_file("branching-test")
+    # Write first row with column names
+    csv_column_names = ['file', 'graph1', 'graph2', 'are_isomorph processing time (s)', 'result', 'amount_of_isomorphisms processing time (s)', 'result']
```

Comment:
Misschien zou ik hier 'passed' van maken ipv 'result'. Zoals ik het begrijp zet je in die kolom of de test geslaagd is, en zo zou je het ook kunnen lezen alsof de grafen isomorph zijn.